### PR TITLE
Initial development

### PR DESCRIPTION
**Pull request by heaper**
Plugin will:

- By default, copy styleguide assets from installed styleguide folder in node_modules to the public folder and run the Pattern Lab build() method

- If plugin is invoked by the file watcher, it will only run the Pattern Lab patternsonly() method

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,84 @@
...
+  "cleanPublic" : true,
+  "defaultPattern": "all",
+  "defaultShowPatternInfo": false,
+  "ishControlsHide": {
+    "s": false,
+    "m": false,
+    "l": false,
+    "full": false,
+    "random": false,
+    "disco": false,
```
Comment (scheul93) [inline README.md]: Consider turning disco on by default

File: __mocks__/patternlab-node.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+//const patternlab = jest.genMockFromModule('patternlab-node');
```
Comment (scheul93) [inline __mocks__/patternlab-node.js]: Consider cleaning up this line, if there aren't plans to use it in the future

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "skeletor-plugin-patternlab",
```
Comment (scheul93) [inline package.json]: Can we change this to include the new scoped package naming conventions. so `@skeletor/plugin-patternlab`

File: __mocks__/patternlab-node.js (hunk 2 above)
Comment (heaper) [reply __mocks__/patternlab-node.js]: ADDRESSED

File: index.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+	logger.error(e);
+	return buildErrorStatusObj(e);
+}
+
+function run(config, options) {
+
+	const buildPatternsOnly = options.source ? true : false;
+
+	const promise = buildPatternsOnly ?
+		build(config, true) :
```
Comment (scheul93) [inline index.js]: Instead of passing `true`, could you pass the `buildPatternsOnly` variable here and on line 50?

File: index.test.js [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+beforeEach(() => {
+	jest.restoreAllMocks();
+
+	patternlab.mockClear();
+
+	patternlab.__reset();
+});
+
+describe('run() returns an error status object', () => {
```
Comment (scheul93) [inline index.test.js:99]: Good job grouping the tests into smaller suites!

**Pull request merged**